components/Ripple/docs/ripple-migration.md

```diff
@@ -0,0 +1,55 @@
+Ink and Ripple provide similar APIs: a view (`MDCInkView`, `MDCRippleView`), and a touch controller (`MDCInkTouchController`, `MDCRippleTouchController`). 
+
+While Ripple and Ink’s implementations slightly vary, the public APIs are nearly identical. Furthermore, Ripple’s API does not produce additional side effects and can be used interchangeably.
+
+These two notions make the migration path from Ink to Ripple relatively simple.
+
+For guidance, these are the current naming differences that you need to pay attention to when migrating:
+
+**`MDCInkView` vs `MDCRippleView`:**
+
+|`MDCInkView`|`MDCRippleView`|
+|---|---|
+|`animationDelegate`|`rippleViewDelegate`|
+|`maxRippleRadius`|`maximumRadius`|
+|`startTouchBeganAtPoint:animated:completion:`|`beginRippleTouchDownAtPoint:animated:completion:`|
+|`startTouchEndAtPoint:animated:completion:`|`beginRippleTouchUpAnimated:completion:`|
+|`inkAnimationDidStart:inkView`|`rippleTouchDownAnimationDidBegin:rippleView`|
+|`inkAnimationDidEnd:inkView`|`rippleTouchUpAnimationDidEnd:rippleView`|
+
+*Other API differences are simply a regex replacement of: (1)Ink(2) to (1)Ripple(2) where (1) and (2) are [a-zA-Z]\*.*
```

Comment:
Ideally a reader of this migration guide will be armed with all information and tools they need to perform the migration. In this case, please provide an exhaustive list of the symbols that map via these regexes.